### ISSUE TITLE
Car parking solution not working, not showing the inference and not counting the occupied and avaible parking spots

### ISSUE THREAD

**Issue by Rasantis** [ultralytics/ultralytics#22435]
### Search before asking

- [x] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and found no similar bug report.

### Bug

i´m using this version:
ultralytics==8.3.217

### Environment

(venv) PS D:\Users\rafa2\OneDrive\Desktop\monitoramento_vaga> yolo checks
Ultralytics 8.3.217  Python-3.11.4 torch-2.9.0+cpu CPU (13th Gen Intel Core(TM) i5-13450HX)
Setup complete  (16 CPUs, 15.7 GB RAM, 272.1/953.9 GB disk)

OS                     Windows-10-10.0.26200-SP0
Environment            Windows
Python                 3.11.4
Install                pip
Path                   D:\Users\rafa2\OneDrive\Desktop\monitoramento_vaga\venv\Lib\site-packages\ultralytics        
RAM                    15.69 GB
Disk                   272.1/953.9 GB
CPU                    13th Gen Intel Core(TM) i5-13450HX
CPU count              16
GPU                    None
GPU count              None
CUDA                   None

numpy                   2.2.6>=1.23.0
matplotlib              3.10.7>=3.3.0
opencv-python           4.12.0.88>=4.6.0
pillow                  12.0.0>=7.1.2
pyyaml                  6.0.3>=5.3.1
requests                2.32.5>=2.23.0
scipy                   1.16.2>=1.4.1
torch                   2.9.0>=1.8.0
torch                   2.9.0!=2.4.0,>=1.8.0; sys_platform == "win32"
torchvision             0.24.0>=0.9.0
psutil                  7.1.0
polars                  1.34.0
ultralytics-thop        2.0.17>=2.0.0
(venv) PS D:\Users\rafa2\OneDrive\Desktop\monitoramento_vaga> 

### Minimal Reproducible Example

i´m using the same code that is in the documentation

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by UltralyticsAssistant** [comment]
👋 Hello @Rasantis, thank you for your interest in Ultralytics 🚀! This is an automated response—an Ultralytics engineer will follow up to assist you soon.

We recommend a visit to the [Docs](https://docs.ultralytics.com/) for new users where you can find many [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) usage examples and where many of the most common questions may already be answered.

If this is a 🐛 Bug Report, please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example/) to help us debug it. Since you mentioned “using the same code that is in the documentation,” please share:
- The exact script/CLI command you’re running
- Model and weights used (e.g., YOLO11n/yolo11n.pt) and any custom configs
- A short sample of input data (or a link to a small subset) and expected vs. actual behavior
- Complete console logs and screenshots of the non-displaying inference output
- Steps to reproduce from a clean environment

If this is a custom training ❓ Question, please provide as much information as possible, including dataset image examples and training logs, and verify you are following our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/).

Join the Ultralytics community where it suits you best. For real-time chat, head to [Discord](https://discord.com/invite/ultralytics) 🎧. Prefer in-depth discussions? Check out [Discourse](https://community.ultralytics.com/). Or dive into threads on our [Subreddit](https://www.reddit.com/r/Ultralytics/) to share knowledge with the community.

## Upgrade

Upgrade to the latest `ultralytics` package including all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) in a **Python>=3.8** environment with **PyTorch>=1.8** to verify your issue is not already resolved in the latest version:

```bash
pip install -U ultralytics
```

## Environments

YOLO may be run in any of the following up-to-date verified environments (with all dependencies including CUDA/CUDNN, Python and PyTorch preinstalled):

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) tests are currently passing. CI tests verify correct operation of all YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) on macOS, Windows, and Ubuntu every 24 hours and on every commit.

**Comment by Y-T-G** [comment]
Did you correctly mark the locations of the spots?

https://docs.ultralytics.com/guides/parking-management/#parking-management-system-code-workflow
